Forecasting and Estimation in Groups, with Confidence.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+>Working from the assumption that groups often make more accurate judgments than individuals, researchers had participants make initial independent estimates of uncertain quantities, discuss their answers in small groups, and revise their estimates.
+
+
+# References
+>Silver, I., Mellers, B., Tetlock, P. (Forthcoming). Wise teamwork: Collective confidence calibration predicts the effectiveness of group discussion. 
+>
+> https://osf.io/tkgea/?view_only=b576f44d1bd74d2f9a36a248855324d9
+>
+> *Ask me for a link to the paper when youre ready to review
+# Stimuli
+## The visual components
+> None given, but below is a list of questions participants answered 
+> ![Alt text](/images/Forcasting_w_confidence_questions.jpg)
+
+
+
+# Procedure
+## Steps
+>Experiment 1:
+> 1. Cell Phones taken from participants
+> 2. Participants are randomly assigned into groups of 3-6; given 2 minutes to introduce themselves
+> 3. Participants then answered an estimation question
+> 4. Then rated their confidence and the difficulty of the question on a scale of 1-7
+> 5. Then participants discussed the question with their groups and tried to improve their answer
+> 6. Answered the same question as before
+> 7. Repeat steps 3-6 for all questions
+> 8. Participants indicated if the group interation improved their second estimates on a scale from -2 to 2.
+>
+> Experiment 2:
+> Same as above but added additional survey questions.
+> 
+> In step 4, researchers asked participants if they expect to get a lot of help from group discussion.
+> 
+> At step 8, participants were asked to identify their most accurate group member. 
+## Roles 
+> Every person had the same role
+
+## Instructions
+> None given, can ask barb
+
```

Comment:
Should we add the stimuli/questions depending on what we get from Barb? 
Study One: 
- estimations of eight country populations estimated in millions of residents from 1-1,000
- twelve historical dates (invention of the printing press) dates between 1000-2018 
Study Two: 
-  4 Questions on historical date questions
- 4 Questions on direct distances between cities like Reykjavik to Nairobi distances were estimated in miles from 1-10,000
-  4 Questions on stock prices for public companies like Twitter and Chipotle estimates were in dollars from 1-500

Reply:
I see you have the question image included- not sure if we need this then!